blazor/accumulation-chart/legend.md

```diff
@@ -293,6 +293,47 @@ When the legend items exceed legend bounds, paging will be enabled by default. E
 
 ![Paging for Legend in Blazor Accumulation Chart](images/legend/blazor-accumulation-chart-legend-paging.png)
 
+### Legend Paging Customization
+
+The Legend Page number Arrow Size can be customized by using ArrowSize API and the page number can be cutomized using TextStyle.
```

Comment:
The legend paging arrow elements can be customized by using the `ArrowSize` property in the `AccumulationChartLegendPageSettings`, and the page numbers can be customized using `AccumulationChartLegendPageSettingsTextStyle`.

Avoid spelling mistakes.